Kiosk/App/UIStoryboardSegueExtensions.swift

```diff
@@ -1,8 +1,7 @@
 import UIKit
 
-extension UIStoryboardSegue {
-}
+extension UIStoryboardSegue {}
 
-func ==(lhs: UIStoryboardSegue, rhs: SegueIdentifier) -> Bool {
+func == (lhs: UIStoryboardSegue, rhs: SegueIdentifier) -> Bool {
```

Comment:
Maybe don't want a space here?

Reply:
Yeah, probably want to handle the cases of `==`, `+=` etc uniquely when used as an override function 👍

Reply:
There's [a rule in SwiftLint](https://github.com/realm/SwiftLint/blob/master/Rules.md#operator-function-whitespace) that actually forces you to always add a trailing space when defining operators.

I guess mostly to have a single rule to avoid Brainfuck-like syntax here:

```
func <|(lhs: Int, rhs: Int) -> Int {}
func <|<<A>(lhs: A, rhs: A) -> A {}
```

Which of these would you prefer?

1. Never add space.
2. Always add space.
3. Whitelist/Blacklist/Heuristic on whether to add whitespace.

Reply:
As that rule is enabled by default in SwiftLint - I'd go with that - keep the space

Reply:
Yeah, I like it aesthetically too.